account_manager/src/common.rs

```diff
@@ -10,6 +10,8 @@ use std::path::{Path, PathBuf};
 /// 62**48 is greater than 255**32, therefore this password has more bits of entropy than a byte
 /// array of length 32.
 const DEFAULT_PASSWORD_LEN: usize = 48;
+const NEWLINE_SLASH_N: u8 = 10;
```

Comment:
It'll be neater if we use Rust's support for character byte literals, rather than hardcoding the ASCII codepoint. Newline is `b'\n'` and carriage return is `b'\r'`.